### PR TITLE
respect CamelCase for enums and their values

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: doc/api_break.dox [hunk 1]
```diff
@@ -1037,6 +1037,13 @@ QgsEditorWidgetFactory         {#qgis_api_break_3_0_QgsEditorWidgetFactory}
 - `alignmentFlag` has been removed. Use QgsFieldFormatter::representValue() instead
 - `createCache` has been removed. Use QgsFieldFormatter::representValue() instead
 
+
+QgsGeometryUtils         {#qgis_api_break_3_0_QgsGeometryUtils}
+----------------
+
+- componentType enum has been renamed to ComponenType and its members were CamelCased too: VERTEX, RING and PART become Vertex, Ring and Part, respectively.
```
Comment (nyalldawson) [inline doc/api_break.dox]: ComponentType

**Review by nyalldawson — APPROVED** [review]
Nice! Was this manual or did you use some tool?

**Comment by 3nids** [comment]
a multiline regex with silver search to find them and manually fixing them using regexxer/qt creator

**Comment by 3nids** [comment]
merged in 4467487

**Pull request closed without merging**